appmail/models.py

```diff
@@ -183,7 +184,7 @@ def clean(self):
     def render_subject(self, context, processors=CONTEXT_PROCESSORS):
         """Render subject line."""
         ctx = Context(helpers.patch_context(context, processors))
-        return Template(self.subject).render(ctx)
+        return unescape(Template(self.subject).render(ctx))
```

Comment:
From Darian offline:

Will marking context safe work? It's a bit backwards, because we escape -> unescape.

https://stackoverflow.com/a/3138985
```
from django.utils.safestring import mark_safe
{key: mark_safe(value) for key, value in context.items()}
```

Reply:
@chisler, you were right. I was marking it safe at the wrong level. This should work now.